cGame_logic.cpp

```diff
@@ -706,16 +704,9 @@ bool cGame::setupGame() {
 
     // TODO: load eventual game settings (resolution, etc)
 
-    // Logbook notification
-    logger->logHeader("Allegro");
-
-    // ALLEGRO - INIT
-    if (allegro_init() != 0) {
-        logger->log(LOG_FATAL, COMP_ALLEGRO, "Allegro init", allegro_id, OUTC_FAILED);
-        return false;
-    }
-
-    logger->log(LOG_INFO, COMP_ALLEGRO, "Allegro init", allegro_id, OUTC_SUCCESS);
+    // FIXME: eventually, we will want to grab this object in the constructor. But then cGame cannot be a
+    // global anymore, because it needs to be destructed before main exits.
+    _PLInit = std::make_unique<cPlatformLayerInit>("d2tm.cfg");
 
     int r = install_timer();
```

Comment:
would it make sense to also move all the other `install_` functions from allegro into the platform class? Perhaps in another PR to keep changes small.

Reply:
I was thinking the same. 
Not sure where the timers should go yet.
Perhaps we can look at how SDL does timers and find some common facade to hide them behind.